models/mpas_atm/model_mod.f90

```diff
@@ -1167,21 +1185,31 @@ subroutine model_interpolate(state_handle, ens_size, location, obs_type, expecte
    goto 100
 endif
 
+if (debug > 1 .and. do_output()) &
+   print *, 'model_interpolate for obs_kind:',&
+             trim(get_name_for_quantity(obs_kind)),' at ',trim(locstring),' cellid:',cellid
+
+! pass surface variables for rttov - it should be ok as these are diagnostic (except for surface elevation).
+if((obs_kind == QTY_SURFACE_PRESSURE) .or. (obs_kind == QTY_SURFACE_ELEVATION)    .or. &
```

Comment:
now that the rttov code passes in a valid value for the vertical location for VERTISSURFACE interpolations, i believe that all of these except surface elevation could be removed from the "special case" code which avoids the test for the difference between the location and the model elevation at that location.  but in the interests of making small steps, feel free to put this on the "phase 2" code changes if you want.

Reply:
putting this into an issue for mpas

Reply:
issue #96